scripts/simple-example.R

```diff
@@ -72,13 +72,16 @@ fit
 prev_plot <- plot_prev(
   fit$summary[[1]], fit$samples[[1]][sample <= 100],
   joint_data$prevalence[[1]]
-)
+) +
+  scale_y_continuous(tran = scales::logit_trans())
```

Comment:
not completely convinced but happy to be persuaded

Reply:
I don't know if it makes sense for presenting to other people but in terms of seeing what is happening without this its really hard to know what is going on at any other time apart from now. Maybe not having percentages is needed if doing this as it make the plot look quite odd.